common/app/views/fragments/media/video.scala.html

```diff
@@ -1,8 +1,10 @@
-@(player: model.VideoPlayer, enhance: Boolean, showEndSlate: Boolean = true, amp: Boolean = false, showPoster: Boolean = true, showOverlay: Boolean = false, seriesTag: Option[model.Tag] = None)
+@(player: model.VideoPlayer, enhance: Boolean, showEndSlate: Boolean = true, amp: Boolean = false, showPoster: Boolean = true, showOverlay: Boolean = false, seriesTag: Option[model.Tag] = None, pressedContent: Option[model.pressed.PressedContent] = None)(implicit requestHeader: RequestHeader)
```

Comment:
This is a bit cryptic. If this generic media video fragment receives a `PressedContent` value, then we render a totally different kind of headline. Currently, `VideoPlayer` holds (nearly all of) the model data, so maybe we should modify that to hold all the data, and the remaining parameters are well-named booleans that control rendering variations.